src/main/java/org/commcare/formplayer/application/MenuController.java

```diff
@@ -63,6 +64,19 @@ public class MenuController extends AbstractBaseController {
 
     private final Log log = LogFactory.getLog(MenuController.class);
 
+    @RequestMapping(value = Constants.URL_INTERVAL_SYNC_DB, method = RequestMethod.POST)
+    @UserLock
+    @UserRestore
+    @AppInstall
```

Comment:
Does this really need an app install?

Reply:
Ya I think so because I need to access `StorageProperty` via `isRestoreXmlExpired` -> `useAggressiveSyncTiming` and from what I understand, that's only accessible with an app install https://github.com/dimagi/formplayer/blob/2cbafaba88f89f6c20a097456da7ee6cb28d37a1/src/main/java/org/commcare/formplayer/services/RestoreFactory.java#L387-L393